AmazonChimeSDK/AmazonChimeSDKTests/audiovideo/data/transcript/TranscriptSpeakerTests.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  TranscriptSpeakerTests.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+@testable import AmazonChimeSDK
+import XCTest
+
+class TranscriptSpeakerTests: XCTestCase {
```

Comment:
I guess there is `AttendeeInfo` so no need?

Reply:
Will remove this class and its test case